src/plugins/discover/public/application/discover_router.test.tsx

```diff
@@ -196,7 +199,7 @@ describe('DiscoverRouter', () => {
         services={mockDiscoverServices}
         history={history}
         profileRegistry={profileRegistry}
-        customizationContext={mockCustomizationContext}
+        customizationContext$={mockCustomizationContext$}
```

Comment:
I couldn't extract the observable value here already, because this is not a React component even though it looks like one.